group-generators/generators/rhinofi-power-users/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import { formatUsers } from ".";
```

Comment:
Can you remove this file?
We have a CI on GitHub that check if the group is well generated or not

Reply:
Done.